packages/babel-parser/src/tokenizer/index.js

```diff
@@ -144,11 +145,9 @@ export default class Tokenizer extends ParserErrors {
   // Move to the next token
 
   next(): void {
-    if (!this.isLookahead) {
-      this.checkKeywordEscapes();
-      if (this.options.tokens) {
-        this.pushToken(new Token(this.state));
-      }
+    this.checkKeywordEscapes();
```

Comment:
The `isLookahead` condition branch is removed because we call `nextToken` in `lookahead()`.